### ISSUE TITLE
Feature: `put` passes the key of the created/updated record to callback

### ISSUE THREAD

**Issue by glsignal** [alekseykulikov/treo#20]
I've made use of this behaviour in the vanilla IndexedDB interface and found it very useful.

When creating/updating records on an object store using `IDBObjectStore.put`, the `onsuccess` event handler will be [passed the record's key as result](https://developer.mozilla.org/en-US/docs/Web/API/IDBObjectStore.put):

```javascript
objectStore.put(record).onsuccess = function(evt) {
  console.log(evt.target.result); // record's key
};
```

It would be helpful, for myself at least, if treo implemented this behaviour perhaps something like this:

```javascript
treoStore.put(record, function(key) {
  console.log(key); // record's key
});
```

What are your thoughts?

**Comment by alekseykulikov** [comment]
Agreed that it's useful. Especially with `autoIncrement`, will be implemented in 0.4.0

**Comment by glsignal** [comment]
:+1: Excellent :)
